config/constants.yml

```diff
@@ -10,11 +10,15 @@
 TERMINUS_VERSION:       '0.12.0'
 
 # Connectivity
-TERMINUS_HOST:          'dashboard.pantheon.io'
+TERMINUS_HOST:          'terminus.pantheon.io'
 TERMINUS_PORT:          443
 TERMINUS_PROTOCOL:      'https'
 TERMINUS_SSH_HOST:      null
 
+# User Prompts
+DASHBOARD_HOST:         'dashboard.pantheon.io'
```

Comment:
I changed it for the prompt that tells you where to go to create a machine token. I missed the `site dashboard` command. I'll search again, but anywhere else it links to something on the Dashboard dashboard?